Author: Arturo Borrero Gonzalez <arturo.borrero.glez@gmail.com>
Description: print verbose output.
 .
 This is a Debian specific patch, no need to forward upstream.
Forwarded: not-needed
Last-Update: 2014-05-15
--- a/Makefile.rules.in
+++ b/Makefile.rules.in
@@ -93,4 +93,4 @@
 .PHONY: $(SUBDIRS)
 $(SUBDIRS):
 			@echo -e "  SUBDIR\t$@/"
-			@$(MAKE) -s -f Makefile.rules $(MAKECMDGOALS) SUBDIR="$@/" SUBDIRS=""
+			@$(MAKE) -f Makefile.rules $(MAKECMDGOALS) SUBDIR="$@/" SUBDIRS=""
